Lib/locale.py

```diff
@@ -321,6 +327,11 @@ def delocalize(string):
         string = string.replace(dd, '.')
     return string
 
+def localize(string, grouping=False, monetary=False):
+    "Parses a string as locale number according to the locale settings"
```

Comment:
Also make this a docstring

Reply:
I do not understand, it is already a docstring.

Reply:
I believe docstrings are supposed to be triple-double-quoted. See https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

Reply:
Well, I follow style of this file.

Reply:
Seems like the majority of the functions in this file have triple-double-quoted docstrings; the single double quotes in `delocalize`, `atof`, `atob` seem to be a mistake.